docroot/themes/humsci/su_humsci_admin/scss/humsci_admin.node_form.scss

```diff
@@ -0,0 +1,18 @@
+.layout-region-node-footer {
```

Comment:
missing charset at the top of the SCSS file. `@charset 'UTF-8';`

Reply:
can you explain why this is needed?

Reply:
Best practices for the most part:
* https://developer.mozilla.org/en-US/docs/Web/CSS/@charset

For SASS, it tells the compiler what to use for character encoding.

Reply:
https://www.sassmeister.com/gist/9537678

If I remove the `content` part the charset goes away. I suspect the compiler adds it when needed.

Reply:
oh thats interesting.